docs/sql/functions/nested.md

```diff
@@ -10,62 +10,67 @@ In the descriptions, `l` is the three element list `[4, 5, 6]`.
+| `array_extract(`*`list`*`, `*`index`*`)`                      | Alias for `list_extract`.                                                                                                                                                          | `array_extract(l, 3)`                    | `6`               |
+| *`list`*`[`*`begin`*`:`*`end`*`]`                             | Bracket notation with colon is an alias for `list_slice`.                                                                                                                          | `l[2:3]`                                 | `[5, 6]`          |
+| *`list`*`[`*`begin`*`:`*`end`*`: `*`step`*`]`                 | `list_slice` in bracket notation with an added `step` feature.                                                                                                                     | `l[:-:2]`                                | `[4, 6]`          |
+| `list_slice(`*`list`*`, `*`begin`*`, `*`end`*`)`              | Extract a sublist using slice conventions. Negative values are accepted. See [slicing](nested#slicing).                                                                            | `list_slice(l, 2, 3)`                    | `[5, 6]`          |
+| `list_slice(`*`list`*`, `*`begin`*`, `*`end`*`, `*`step`*`)`  | `list_slice` with added `step` feature.                                                                                                                                            | `list_slice(l, 1, 3, 2)`                 | `[4, 6]`          |
+| `array_slice(`*`list`*`, `*`begin`*`, `*`end`*`)`             | Alias for `list_slice`.                                                                                                                                                            | `array_slice(l, 2, 3)`                   | `[5, 6]`          |
+| `array_slice(`*`list`*`, `*`begin`*`, `*`end`*`, `*`step`*`)` | Alias for `list_slice` with `step`.                                                                                                                                                | `array_slice(l, 1, 3, 2)`                | `[4, 6]`          |
+| `list_reverse(`*`list`*`)`                                    | Reverses the list.                                                                                                                                                                 | `list_reverse(l)`                        | `[6, 5, 4]`       |
+| `array_reverse(`*`list`*`)`                                   | Alias for `list_reverse`.                                                                                                                                                          | `array_reverse(l)`                       | `[6, 5, 4]`       |
```

Comment:
Can we add an alias column instead of double listing functions?

Reply:
Agreed, it makes the table cluttered.